ExampleService/ExampleService_WebApi/Controllers/TaskController.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExampleService_WebApi;
+
+[ApiController]
+[Route("[controller]")]
+public class TaskController :  ControllerBase
+{   
+    private readonly InterfaceClass _interface;
+
+    //injecting the interface dependency
+    public TaskController(InterfaceClass interfaceClass)
+    {
+        _interface= interfaceClass;
+    }
+
+    //to return the list of task
+    [HttpGet("List of your Task")]
+    public IActionResult Get()
+    {   
+        var taskList = _interface.ListofTask();
+
+        //mapping as DTO
+        List<TaskDTO> taskMapping = new List<TaskDTO>();        
+
+        foreach(var tasks in taskList)
+        {
+            var temp = new TaskDTO();
+            temp.TaskDescription = tasks.TaskDescription;
+            temp.TaskName = tasks.TaskName;
```

Comment:
This mapper is used in other API also, in these cases we can create a reusable method to avoid repeating the same code in many places.

Reply:
Since they are used for different logic, I could not find a reusable code.